app/src/main/java/com/appunite/loudius/ui/authenticating/AuthenticatingScreen.kt

```diff
@@ -22,9 +22,10 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.hilt.navigation.compose.hiltViewModel
 import com.appunite.loudius.R
-import com.appunite.loudius.ui.components.LoudiusFullScreenError
+import com.appunite.loudius.screenshots.R.string
```

Comment:
I'd suggest instead of importing `R.string` I'd use:
```
import com.appunite.loudius.screenshots.R as screenshotsR`
```

Reply:
Good idea 👌